nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/RefManager.java

```diff
@@ -246,11 +246,7 @@ String getPropValue(int dim, String propName, boolean getDefault) {
                     }
                     break;
                 case "acqarray":
-                    if (getDefault) {
-                        value = "0";
-                    } else {
-                        value = "0";
-                    }
+                    value = "0";
```

Comment:
I am unsure on this one whether I should have removed of fixed the if/else. In relation to this on Line 360 (https://github.com/nanalysis/nmrfx/blob/7d9b8e15b120fd9ddb133501a913dcd9801e67fe/nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/RefManager.java#L360), the default value comment is hardcoded to 0, should the default value be retrieved by the line `String defaultValue = getPropValue(dim, probName, true);` ? It also seems that getPropValue is never really called for “acqarray” since the other place it might have been called avoids calling it (https://github.com/nanalysis/nmrfx/blob/7d9b8e15b120fd9ddb133501a913dcd9801e67fe/nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/RefManager.java#L172)